packages/atomic/stencil.config.ts

```diff
@@ -113,9 +113,9 @@ export const config: Config = {
       plugins: [
         atImport(),
         mixins(),
+        tailwindNesting(),
```

Comment:
We have to use `tailwindcss/nesting` & put it before `tailwind()` for it to work, we get errors in the other case

Reply:
With this, we are getting an issue with mixins & nesting, the following doesn't work (same pattern as used in result templates):
```css
/* Mixin with nesting */
@define-mixin test-mixin {
  input {
    background-color: red;
  }
}

div {
  @mixin test-mixin;
}
```

The fix is to reuse `postcssNesting()` after `tailwind()` like we did before.